AppController/djinn.rb

```diff
@@ -879,10 +880,14 @@ def check_options(options)
       # won't be able to translate it.
       unless (val.class == String or val.class == PARAMETERS_AND_CLASS[key][0] or
          (PARAMETERS_AND_CLASS[key][0] == TrueClass and val.class == FalseClass))
-        begin
-          msg = "Removing parameter '" + key + "' with unknown value '" +\
-            val.to_s + "'."
-        rescue
+        if PARAMETERS_AND_CLASS[key][2]
```

Comment:
It would be nice to turn 0, 1, and 2 into named index constants now that the PARAMETERS_AND_CLASS has so many values per key.

It would make it easier to understand what each field represents.